webflow-hacks/misc-hacks/54/54.ts

```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const CHECKBOX_FIELD_SELECTOR = '[fs-hacks-element="checkbox-field"]';
+  const CHECKBOX_LABEL_SELECTOR = '.w-form-label';
```

Comment:
Can we remove the `CHECKBOX_LABEL_SELECTOR` constant in this case? We don't want people to think they can update it, as it's a Webflow internal class.

Reply:
okay